### PR TITLE
feat: update external api

### PR DESCRIPTION
**Pull request by stepanLav**
New endpoints tested:
- <details><summary>Karura ❌ (Was removed from that PR, SubQuery team is working on fix that)</summary> {"code": 2000,"error": "Request failed: FetchError: no available order. retries: 0."} </details> 
- <details><summary>Shiden ✅</summary> Screenshot 2024-09-24 at 10 17 26</details>
- <details><summary>Bifrost ✅</summary> Screenshot 2024-09-24 at 10 17 47</details>
- <details><summary>Basilisk ✅</summary> Screenshot 2024-09-24 at 10 18 10</details>
- <details><summary>Altair ✅</summary> Screenshot 2024-09-24 at 10 18 28</details>
- <details><summary>Khala ✅</summary> Screenshot 2024-09-24 at 10 18 50 </details>
- <details><summary>Kilt ✅</summary> Screenshot 2024-09-24 at 10 20 31 </details>
- <details><summary>Calamari ✅</summary> Screenshot 2024-09-24 at 10 21 00</details>
- <details><summary>Quartz ✅</summary> Screenshot 2024-09-24 at 10 21 31</details>
- <details><summary>Bit.Country ✅</summary> Screenshot 2024-09-24 at 10 22 02 </details>
- <details><summary>Acala ✅</summary> Screenshot 2024-09-24 at 10 22 30 </details>
- <details><summary>PAH ✅</summary> Screenshot 2024-09-24 at 10 23 07 </details>
- <details><summary>Picasso ✅</summary> Screenshot 2024-09-24 at 10 23 41 </details>
- <details><summary>Polkadex ✅</summary> Screenshot 2024-09-24 at 10 24 17</details>

Postman collection was created - [link](https://grey-meadow-881226.postman.co/workspace/NOVASAMA~a5cbe27c-5c3a-4a59-a38e-b9e75cd33290/collection/4781508-c63cee7f-c0d9-4622-8fe4-86d42c046dbe?action=share&creator=4781508)

### CODE REVIEW

File: chains/v20/chains.json [hunk 1]
```diff
@@ -2921,13 +2945,22 @@
...
+                    "url": "https://gateway.subquery.network/query/0x36"
+                }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "calamari"
+                    }
```
Comment (stepanLav) [inline chains/v20/chains.json:2963]: Not in production
```suggestion
```

File: chains/v20/chains.json [hunk 2]
```diff
@@ -2921,13 +2945,14 @@
...
-                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-calamari"
+                    "url": "https://gateway.subquery.network/query/0x36"
                 }
             ],
             "staking": [
                 {
                     "type": "subquery",
-                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-calamari"
+                    "url": "https://gateway.subquery.network/query/0x36"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:2955]:
```suggestion
```

File: chains/v20/chains.json [hunk 3]
```diff
@@ -3551,6 +3575,15 @@
...
                 }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "astar"
+                    }
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:3586]: not in prod
```suggestion
```

File: chains/v20/chains.json [hunk 4]
```diff
@@ -2736,7 +2751,16 @@
...
+                    "url": "https://gateway.subquery.network/query/0x3a"
+                }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "khala"
+                    }
```
Comment (stepanLav) [inline chains/v20/chains.json:2763]: not in prod
```suggestion
```

File: chains/v20/chains.json [hunk 5]
```diff
@@ -1886,7 +1886,16 @@
...
+                    "url": "https://gateway.subquery.network/query/0x40"
+                }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "shiden"
+                    }
```
Comment (stepanLav) [inline chains/v20/chains.json:1898]: not in prod
```suggestion
```

File: chains/v20/chains.json [hunk 6]
```diff
@@ -2123,7 +2123,13 @@
...
                 {
                     "type": "subquery",
-                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-bifrost"
+                    "url": "https://gateway.subquery.network/query/0x38"
+                }
+            ],
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://gateway.subquery.network/query/0x38"
```
Comment (stepanLav) [inline chains/v20/chains.json:2132]: not in prod
```suggestion
```

File: chains/v20/chains.json [hunk 7]
```diff
@@ -3778,6 +3778,15 @@
...
                 }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "parallel"
+                    }
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:3789]: not in prod
```suggestion
```

File: chains/v20/chains.json [hunk 8]
```diff
@@ -4622,6 +4622,15 @@
...
                 }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "crustshadow"
+                    }
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:4633]:
```suggestion
```

File: chains/v20/chains.json [hunk 9]
```diff
@@ -5866,6 +5866,12 @@
                     "type": "subquery",
                     "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-turing"
                 }
+            ],
+            "governance": [
+                {
+                    "type": "subsquare",
+                    "url": "https://turing.subsquare.io/api"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:5874]:
```suggestion
```

File: chains/v20/chains.json [hunk 10]
```diff
@@ -6149,13 +6158,22 @@
...
+                    "url": "https://gateway.subquery.network/query/0x20"
+                }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "polkadex"
+                    }
```
Comment (stepanLav) [inline chains/v20/chains.json:6176]:
```suggestion
```

File: chains/v20/chains.json [hunk 11]
```diff
@@ -7003,6 +7021,15 @@
...
                 }
+            ],
+            "governance": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://polkassembly-hasura.herokuapp.com/v1/graphql",
+                    "parameters": {
+                        "network": "amplitude"
+                    }
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:7032]:
```suggestion
```

File: chains/v20/chains.json [hunk 12]
```diff
@@ -8331,6 +8366,12 @@
             }
         ],
         "externalApi": {
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet---joystream"
+                }
+            ],
```
Comment (stepanLav) [inline chains/v20/chains.json:8374]:
```suggestion
```

File: chains/v20/chains.json [hunk 13]
```diff
@@ -8749,6 +8790,12 @@
                     "type": "subquery",
                     "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-continuum"
                 }
+            ],
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-continuum"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:8798]:
```suggestion
```

File: chains/v20/chains.json [hunk 14]
```diff
@@ -8784,7 +8831,21 @@
...
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-enjin-relay"
+                }
+            ],
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-enjin-relay"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:8846]:
```suggestion
```

File: chains/v20/chains.json [hunk 15]
```diff
@@ -8883,6 +8944,20 @@
...
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-acurast"
+                }
+            ],
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-acurast"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:8959]:
```suggestion
```

File: chains/v20/chains.json [hunk 16]
```diff
@@ -8914,6 +8989,12 @@
                     "type": "subquery",
                     "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-kreivo"
                 }
+            ],
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-kreivo"
+                }
```
Comment (stepanLav) [inline chains/v20/chains.json:8997]:
```suggestion
```

File: chains/v20/chains.json [hunk 17]
```diff
@@ -8950,6 +9031,12 @@
             }
         ],
         "externalApi": {
+            "staking": [
+                {
+                    "type": "subquery",
+                    "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-hyperbridge"
+                }
+            ],
```
Comment (stepanLav) [inline chains/v20/chains.json:9039]:
```suggestion
```

**Review by valentunn — APPROVED** [review]

**Review by leohar — APPROVED** [review]

**Pull request merged**